packages/dht/src/identifiers.ts

```diff
@@ -28,3 +28,16 @@ export const createRandomDhtAddress = (): DhtAddress => {
     return getDhtAddressFromRaw(crypto.randomBytes(KADEMLIA_ID_LENGTH_IN_BYTES))
 }
 
+export const serializePeerDescriptorForSigning = (peerDescriptor: PeerDescriptor): Uint8Array => {
+    const undefinedBuffer = Buffer.from('undefined')
+    const buffers = [ 
+        peerDescriptor.type !== undefined ? Buffer.from([peerDescriptor.type]) : undefinedBuffer,
+        peerDescriptor.udp !== undefined ? ConnectivityMethod.toBinary(peerDescriptor.udp) : undefinedBuffer,
+        peerDescriptor.tcp !== undefined ? ConnectivityMethod.toBinary(peerDescriptor.tcp) : undefinedBuffer,
+        peerDescriptor.websocket !== undefined ? ConnectivityMethod.toBinary(peerDescriptor.websocket) : undefinedBuffer,
+        peerDescriptor.region !== undefined ? Buffer.from([peerDescriptor.region]) : undefinedBuffer,
```

Comment:
Can we be sure that region is always one byte value (i.e. less than `256`)

Reply:
no

Reply:
Then we should some other way to convert region to a Buffer. Currently it just gets the last byte: e.g.  `0x1E240: Buffer.from([123456]) -> 0x40`

Reply:
Not sure I understood this point? Let's discuss on call for example.